### DISCUSSION TITLE
What is WinRing0x64.dll and WinRing0x64.sys used for?

### DISCUSSION THREAD

**Discussion by ryan2458** [seerge/g-helper#1087] [Q&A] [answered]
Noticed these appear after one of the updates.  Putting these into google immediately returns results stating that these are used for trojan horses and bitcoin miners.  I take it this isn't the case, so what are they actually being used for?  Is the code open source?

**Comment by seerge** [comment, marked as answer]
@ryan2458 hello. It's a "low level" access [library](https://github.com/GermanAizek/WinRing0), that is needed for AMD CPU Undervolting (app uses library from https://github.com/JamesCJ60/Universal-x86-Tuning-Utility for that, that also in turn uses winring).

If you don't have AMD CPU or don't undervolt those files are not needed.

**Reply by rnjhnd** [reply]
Can i delete it?

**Reply by seerge** [reply]
If you don't use it to undervolt AMD CPU – you can :)
